feconf.py

```diff
@@ -671,6 +676,10 @@ def get_empty_ratings():
 # External URL for the Foundation site.
 FOUNDATION_SITE_URL = 'http://oppiafoundation.org'
 
+# External URL for the oppia production site. Change this for internal testing
+# in testing production environments.
+OPPIA_SITE_URL = 'https://oppia.org'
```

Comment:
Change to 'http://localhost:8181' (for dev) or None, we will switch it during deployment.

Reply:
Done.